tests/foreman/cli/test_activationkey.py

```diff
@@ -73,7 +73,7 @@ def test_positive_create_with_name(module_target_sat, module_entitlement_manifes
 
     :parametrized: yes
     """
-    new_ak = module_target_sat.cli_factory.make_activation_key(
+    new_ak = CLIFactory.CLIFactory.make_activation_key(
```

Comment:
```suggestion
    new_ak = module_target_sat.cli_factory.make_activation_key(
```
The right way of using cli_factory methods.

Reply:
Incorrect commit happened here. I will correct it today.